providers/fips/fipsprov.c

```diff
@@ -375,7 +375,7 @@ static const OSSL_ALGORITHM fips_digests[] = {
     { "KECCAK-KMAC-256:KECCAK-KMAC256", "provider=fips,fips=yes", keccak_kmac_256_functions },
 
     /* Non-FIPS algorithm to support oneshot_hash in the Ed448 code */
-    { "SHAKE-256:SHAKE256", "provider=fips,fips=no", shake_256_functions },
+    { "SHAKE-256:SHAKE256", "provider=fips,fips=yes", shake_256_functions },
```

Comment:
Are we also vendor affirming this one?

Reply:
I think it will need to be validated -- this is a consequence of having Ed448.

At this stage, let's assume it is in.  We can remove it later if the lab disagrees.

Reply:
I've emailed the lab to ask this specific question.